src/Microsoft.Health.Fhir.Core/Features/Search/SearchParameterNotSupportedException.cs

```diff
@@ -45,6 +45,7 @@ public SearchParameterNotSupportedException(string resourceType, string paramNam
         /// Initializes a new instance of the <see cref="SearchParameterNotSupportedException"/> class.
         /// </summary>
         /// <param name="definitionUri">The search parameter definition URL.</param>
+        /// <param name="location">The search parameter location of the error.</param>
```

Comment:
>/// <param name="location">The search parameter location of the error.</param> [](http://example.com/codeflow?start=8&length=78)

this seems off

Reply:
yes missed reverting this back. should be good now